contracts/l1/L1DAIWormholeBridge.sol

```diff
@@ -70,13 +51,11 @@ contract L1DAIWormholeBridge is OVM_CrossDomainEnabled {
     address _escrow,
     address _wormholeRouter
   ) OVM_CrossDomainEnabled(_l1messenger) {
-    wards[msg.sender] = 1;
-    emit Rely(msg.sender);
-
     l1Token = _l1Token;
     l2DAIWormholeBridge = _l2DAIWormholeBridge;
     escrow = _escrow;
     wormholeRouter = WormholeRouter(_wormholeRouter);
+    TokenLike(_l1Token).approve(_wormholeRouter, type(uint256).max);
```

Comment:
Maybe leaving a comment here that wormholeRouter can spent funds from the bridge but NOT from the escrow could reduce confusion while reading code? 🤔